src/GHC/Util.hs

```diff
@@ -15,8 +18,14 @@ module GHC.Util (
 
   , parsePragmasIntoDynFlags
   , parseFileGhcLib, parseExpGhcLib, parseImportGhcLib
+  , hseSpanToGHC, hseLocToGHC
```

Comment:
Is this module an ok location for all these things?

Reply:
I'm a bit reluctant to have the GHC modules depend on HSE if we can get away with it. We've managed to avoid that until now. Note also there are the functions `ghcSpanToHSE`, `ghcSrcLocToHSE` in `HSE/All.hs`.

Reply:
I moved it to `HSE.All`

I'm restating the same question about the other functions.

Reply:
I'm not overly fussed about where the functions are for now. I imagine all this stuff will move over time.

Reply:
Thanks for that. Yes, not a big deal but nice that it worked out putting them in `All`. These conversion functions will ultimately be eliminated 😄